code/datums/mutations/passive.dm

```diff
@@ -0,0 +1,12 @@
+/datum/mutation/human/biotechcompat
```

Comment:
you sure you dont want to set instability or difficulty for this?

Reply:
Default difficulty is fine, i guess i can add some minimal instability.